### DISCUSSION TITLE
Messenger inbox option grayed out

### DISCUSSION THREAD

**Discussion by jaimecoj** [chatwoot/chatwoot#8473] [Self hosted]
I am using self hosted chatwoot at Linode. I added successfully whatsapp & web inboxes, however when I try to select Messenger inbox to connect instagram account it is grayed out. May I know why?

![photo_2023-12-01_12-45-58](https://github.com/chatwoot/chatwoot/assets/9117697/f85d8f42-e0ca-49cb-82a9-f9c7a934f330)

**Comment by mdsami** [comment]
you can add a custom domain to enable it . let me know if you want any help on it!

**Reply by iiihexiii** [reply]
Yes need help with that

**Comment by severinow3** [comment]
Hello, mine is like this too, can anyone help me solve it?

Mine is also self hosted

**Comment by sojan-official** [comment]
You'll needsh the set-up for the messenger channel to be available. 
https://www.chatwoot.com/docs/self-hosted/configuration/features/integrations/facebook-channel-setup/

**Reply by severinow3** [reply]
Thanks.
